tests/api.py

```diff
@@ -1,21 +1,29 @@
+# pylint: disable=superfluous-parens
```

Comment:
😭

Reply:
There are a lot of pylint comments in sheepdog already. It's not just me this time. 😇